.vscode/launch.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
I think that VSC settings should be private for a user, and should not be pushed to the repository.